bootstrap/custom.scss

```diff
@@ -27,7 +27,7 @@ $font-sizes: (
   6: 16px
 );
 
-@import "onboarding-assets/bootstrap-5.0.2/scss/bootstrap.scss";
+@import "/Users/user/Desktop/onboarding/onboarding_eW/bootstrap-5.0.2/scss/bootstrap.scss";
```

Comment:
never use absolute paths! (if I download this in my machine, it's not going to work). Use relative paths (`../`)